src/reconstruction/refinement/convertToIrreversible.m

```diff
@@ -25,88 +29,184 @@
 %       - written by Gregory Hannum 7/9/05
 %       - Modified by Markus Herrgard 7/25/05
 %       - Modified by Jan Schellenberger 9/9/09 for speed.
+%       - Modified by Diana El Assal & Fatima Monteiro 6/2/17 allow to
+%       optionally only split a specific list of reversible reactions to
+%       irreversible, without appending '_r'. In this case, model.rev =
+%       false (forward) and model.rev = true (reversible).
 
-modelIrrev.S = spalloc(size(model.S, 1), 0, 2 * nnz(model.S)); % declare variables
+modelIrrev.S = spalloc(size(model.S,1),0,2*nnz(model.S)); %declare variables
 modelIrrev.rxns = [];
-modelIrrev.lb = zeros(2 * length(model.rxns), 1);
-modelIrrev.ub = zeros(2 * length(model.rxns), 1);
-modelIrrev.c = zeros(2 * length(model.rxns), 1);
-matchRev = zeros(2 * length(model.rxns), 1);
+modelIrrev.rev = zeros(2*length(model.rxns),1);
```

Comment:
@dayena, `model.rev` has been deprecated in PR #507. May you adapt your code accordingly? Thank you.

cc: @tpfau